apps/prairielearn/src/pages/instructorFileBrowser/instructorFileBrowser.html.ts

```diff
@@ -69,159 +176,129 @@ export function InstructorFileBrowser({ resLocals }: { resLocals: Record<string,
             `<%- include('../partials/${syncErrorsPartial}') %>`,
             resLocals,
           )}
-          ${!authz_data.has_course_permission_view
-            ? html`
-                <div class="card mb-4">
-                  <div class="card-header bg-danger text-white">Files</div>
-                  <div class="card-body">
-                    <h2>Insufficient permissions</h2>
-                    <p>You must have at least &quot;Viewer&quot; permissions for this course.</p>
-                    ${course_owners.length > 0
-                      ? html`
-                          <p>Contact one of the below course owners to request access.</p>
-                          <ul>
-                            ${course_owners.map(
-                              (owner) => html`
-                                <li>${owner.uid} ${owner.name ? `(${owner.name})` : ''}</li>
-                              `,
-                            )}
-                          </ul>
-                        `
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              <div class="row align-items-center justify-content-between">
+                <div class="col-auto text-monospace d-flex">
+                  ${joinHtml(
+                    paths.branch.map(
+                      (dir) => html`
+                        ${dir.canView
+                          ? html`
+                              <a
+                                class="text-white"
+                                href="${paths.urlPrefix}/file_view/${dir.encodedPath}"
+                              >
+                                ${dir.name}
+                              </a>
+                            `
+                          : html`<span>${dir.name}</span>`}
+                      `,
+                    ),
+                    html`<span class="mx-2">/</span>`,
+                  )}
+                </div>
+                <div class="col-auto">
+                  ${isFile
+                    ? FileBrowserActions({ paths, fileInfo, csrfToken })
+                    : authz_data.has_course_permission_edit && !course.example_course
```

Comment:
I know this didn't change in this PR, but: do you know why we don't check permissions + example course for `FileBrowserActions`?

Reply:
One of the actions is file download, which does not require edit permission and can be done in the example course.